### ISSUE TITLE
1.9.0+369 not working on Intel MacOS

### ISSUE THREAD

**Issue by MeestorX** [bitfocus/companion-satellite#146]
Tried on Big Sur and Sonoma. Running app just brings up the usual security warning (this file was downloaded from the internet) but nothing else. Double-clicking it a 2nd time does nothing as well.

Where can I find the logs to see what might be stopping it from starting?

**Comment by Julusian** [comment]
you could try launching it from the terminal, to see if it logs anything.
you might need to cd inside the `Companion Satellte.app/Contents/MacOS`, then run it with `./Companion\ Satellite`

**Comment by MeestorX** [comment]
TY. Without an SD connected, I do see the tray icon and can see the config. Terminal shows the following, as I don't have Companion running...
```
Connecting to 127.0.0.1:16622
Error: connect ECONNREFUSED 127.0.0.1:16622
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1606:16) {
  errno: -61,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 16622
}
Connection closed
```
When I connect a SD, Terminal shows:
```
Found a device {
  bLength: 18,
  bDescriptorType: 1,
  bcdUSB: 512,
  bDeviceClass: 0,
  bDeviceSubClass: 0,
  bDeviceProtocol: 0,
  bMaxPacketSize0: 64,
  idVendor: 4057,
  idProduct: 109,
  bcdDevice: 512,
  iManufacturer: 1,
  iProduct: 2,
  iSerialNumber: 3,
  bNumConfigurations: 1
}
adding new device: DevSrvsID:4295007165
existing = []
Registering key events for AL31K2C73422
dyld[10120]: missing symbol called
Killed: 9
logout
```

**Comment by tjcooney** [comment]
im having the same issuie, its not connecting on macos

**Comment by MeestorX** [comment]
@Julusian Any idea what `dyld[10120]: missing symbol called` means?

**Comment by Julusian** [comment]
Sounds like it tried to load/call a method from a c/c++ library, and the method doesnt exist.

I have no idea where to begin on trying to figure out what the method is, or what library that could be from. Especially as it works fine on my mac (M1 mini running Sonoma)

**Comment by MeestorX** [comment]
Wouldn't there be logs somewhere?

**Comment by MeestorX** [comment]
Threre's some discussion that if the library is created for the wrong architecture, this kind of error can appear. So, if satellite is created for Mac-Arm and run on a Mac-Intel, this kind of error can appear. Could you double-check that the Mac-Intel Build is definitely built for Intel?
(My testing was on an Intel Mac)

**Comment by Julusian** [comment]
Ah yep, running the intel build on arm triggers it. I can work with that

**Comment by Julusian** [comment]
Its still building so I haven't verified it, hopefully beta 371 resolves this

**Comment by Julusian** [comment]
the intel build of this now works on arm for me. published as 1.9.1

**Comment by MeestorX** [comment]
Can confirm fixed.
TY!
